packages/console/src/pages/UserDetails/index.tsx

```diff
@@ -55,6 +55,11 @@ const UserDetails = () => {
     return userDetailsParser.toLocalForm(data);
   }, [data]);
 
+  useEffect(() => {
+    setIsDeleteFormOpen(false);
+    setIsResetPasswordFormOpen(false);
```

Comment:
I think you only need to detect if the route has been changed. If changed, close all modals. You don't necessarily have to check if the current route ends with `/logs`

Reply:
Agreed this should be a general fix PR. All the page-level models should close automatically when navigating between pages.  
P.S. the PR title is a little bit misleading as well.

Reply:
updated.